client/utils/account-fees.tsx

```diff
@@ -74,24 +74,27 @@ const getStripeFeeSectionUrl = ( country: string ): string => {
 	);
 };
 
-const getFeeDescriptionString = ( fee: BaseFee ): string => {
+const getFeeDescriptionString = (
+	fee: BaseFee,
+	discountRate: number
+): string => {
 	if ( fee.fixed_rate && fee.percentage_rate ) {
 		return sprintf(
 			'%1$f%% + %2$s',
-			formatFee( fee.percentage_rate ),
-			formatCurrency( fee.fixed_rate, fee.currency )
+			formatFee( fee.percentage_rate * discountRate ),
+			formatCurrency( fee.fixed_rate * discountRate, fee.currency )
 		);
 	} else if ( fee.fixed_rate ) {
 		return sprintf(
 			'%2$s',
-			formatFee( fee.percentage_rate ),
-			formatCurrency( fee.fixed_rate, fee.currency )
+			formatFee( fee.percentage_rate * discountRate ),
+			formatCurrency( fee.fixed_rate * discountRate, fee.currency )
 		);
 	} else if ( fee.percentage_rate ) {
 		return sprintf(
 			'%1$f%%',
-			formatFee( fee.percentage_rate ),
-			formatCurrency( fee.fixed_rate, fee.currency )
+			formatFee( fee.percentage_rate * discountRate ),
```

Comment:
Similar to the comment on line 87-90, Do we need to have `formatCurrency( fee.fixed_rate * discountRate, fee.currency ) within the `sprintf` statement , when we do not expect it? 

```suggestion
else if ( fee.percentage_rate ) {
		return sprintf(
			'%1$f%%',
			formatFee( fee.percentage_rate * discountRate ),
        );
```